docs/deploy.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -e # Exit with nonzero exit code if anything fails
+
+SOURCE_BRANCH="master"
+TARGET_BRANCH="gh-pages"
+
+REPO=$(git config remote.origin.url)
+SHA=$(git rev-parse --verify HEAD)
+HTTPS_REPO=${REPO/https:\/\/github.com\//https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/}
+OUT_DIR="cloned-gh-pages"
+
+echo "Repo: " ${REPO}
+echo "HTTPS_REPO: " ${HTTPS_REPO}
```

Comment:
Can you remove this echo? - We don't want secrets shown in the public travis log